readthedocs/urls.py

```diff
@@ -17,6 +17,9 @@
 handler403 = ErrorView.as_view(status_code=403)
 handler404 = ErrorView.as_view(status_code=404)
 handler500 = ErrorView.as_view(status_code=500)
+# Rate limit handler for allauth views,
+# this isn't a Django built-in handler.
+handler429 = ErrorView.as_view(status_code=429)
```

Comment:
I didn't find this in their docs at https://docs.allauth.org/en/latest/account/rate_limits.html

Where did you find the `handler429` has to be defined? Please, put a link to the docs in this comment.

Reply:
isn't in the docs, yet https://codeberg.org/allauth/django-allauth/pulls/4139/

Reply:
Cool, let's add a link to that PR as a comment for now.

```suggestion
# https://codeberg.org/allauth/django-allauth/pulls/4139
handler429 = ErrorView.as_view(status_code=429)
```